tests/diff-plugin-outputs.js

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/node
+
+const fs = require('fs');
+const path = require('path');
+const del = require('node-delete');
+const minimist = require('minimist');
+const git = require('simple-git')();
+
+const args = minimist(process.argv.slice(2), {
+  string: ['p', 'r'],
+  boolean: 'h',
+  alias: { p: 'plugin', r: 'ref', h: 'help' },
+  default: { r: 'HEAD~1' },
+  unknown: function () { },
+});
+
+if (args.help) {
+  const helpMessage = [
+    'This script compares the output of the given fixtures with another version in the current repository.',
+    `Usage: ${process.argv[1]} <-p plugin> [-r ref] <fixtures>`,
```

Comment:
change this to something like `-p <plugin name> [-r <git reference>] <fixture key> [<more fixture keys>]`